### ISSUE TITLE
Getting device id

### ISSUE THREAD

**Issue by bemindt** [blacksmithlabs/orbit-bhyve-remote#16]
In your notes you mention that you don't know how to get the device id.  If you do a HTTP get to the following url:

                https://api.orbitbhyve.com/v1/devices?user_id=your_user_id

You will get a payload back with a lot of information including the device id for each bhye device you have.  Of course you need to have the appropriate headers set on the get request.

**Comment by killermonk** [comment]
Thanks! This end point is implemented in my hosted firebase solution since it has a UI that lets you build the configuration.

For configuring this headless, self hosted version it doesn't make as much sense to try to call the API during configuration. The manual steps in the readme felt like the best way for this project.

**Comment by bemindt** [comment]
I am working on a java based solution for this and I am trying to model after the work you did, but unable to get the websockets to respond.  I am not a js expert, but are you really using web sockets?

**Comment by killermonk** [comment]
Yes. I am using web sockets to do all the communication. I just kind of built it as needed, so the code is not super well structured or planned out. I've started a discussion where I tried to capture everything I could about the design and handling of the web socket that makes everything work.

Let's take this over there: https://github.com/blacksmithlabs/orbit-bhyve-remote/discussions/17

I've never coded WebSockets in Java, but am quite experience using the language. Feel free to post any code you're writing that is having issues in the discussion and I can help you debug it.
